iOSExampleWithPods/STUNClient.xcodeproj/project.pbxproj

```diff
@@ -291,12 +287,33 @@
 			buildActionMask = 2147483647;
 			files = (
 			);
-			inputFileListPaths = (
-				"${PODS_ROOT}/Target Support Files/Pods-STUNClientTests/Pods-STUNClientTests-frameworks-${CONFIGURATION}-input-files.xcfilelist",
+			inputPaths = (
+				"${PODS_ROOT}/Target Support Files/Pods-STUNClientTests/Pods-STUNClientTests-frameworks.sh",
+				"${BUILT_PRODUCTS_DIR}/CNIOAtomics/CNIOAtomics.framework",
```

Comment:
are you sure we need all those paths to the explicit frameworks in 3 places.

Reply:
We use Development Pods in iOSExampleWithPods.
`pod 'StunClient', :path => '..'`